apps/pkeyutl.c

```diff
@@ -68,7 +68,8 @@
 
 static EVP_PKEY_CTX *init_ctx(int *pkeysize,
                               char *keyfile, int keyform, int key_type,
-                              char *passinarg, int pkey_op, ENGINE *e);
+                              char *passinarg, int pkey_op, ENGINE *e,
+			      const int impl);
```

Comment:
Doh... Yes I certainly can. Thought I've taken care of it, guess I spoke too soon. :-(